src/05-derivatives/01-introduction.md

```diff
@@ -0,0 +1,221 @@
+
+Example:
+
+```JSON
+{
+    "Name": "FMRIPREP Outputs",
+    "BIDSVersion": "TODO (depends when this PR will be merged)",
+    "PipelineDescription": {
+        "Name": "FMRIPREP",
+        "Version": "1.2.5",
+        "DockerHubContainerTag": "poldracklab/fmriprep:1.2.5"
+        },
+    "SourceDatasets": [
+        {
+            "DOI": "10.18112/openneuro.ds000114.v1.0.1",
+            "URL": "https://openneuro.org/datasets/ds000114/versions/1.0.1",
+            "Version": "1.0.1"
+        }
+    ]
+}
+```
+
+## Coordinate systems
+
+The coordinate system (a.k.a. space) a particular derivative is in SHOULD be
+denoted using a filename keyword `space` whenever such keyword is present in
+the filename template of a given derivative type. The allowed values for this
+keyword depend on the file format:
+
+| File format                  | Description             | Allowed `CoordinateSystem` values                                                                                                                                                                             |
+| ---------------------------- | ----------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| NIfTI (`.nii` and `.nii.gz`) | Volume data             | Coordinate systems listed in [Template Based Coordinate Systems: Volume](../99-appendices/08-coordinate-systems.md#volume). If `individual` or `custom` is used then setting `SpatialReference` is REQUIRED   |
+| GIFTI (`.gii`)               | Surface data            | Coordinate systems listed in [Template Based Coordinate Systems: Surface](../99-appendices/08-coordinate-systems.md#surface). If `individual` or `custom` is used then setting `SpatialReference` is REQUIRED |
+| CIFTI (`.nii`)               | Volume and surface data | Coordinate systems listed in [Template Based Coordinate Systems: Hybrid (Volume/Surface) aliases](../99-appendices/08-coordinate-systems.md#hybrid-volumesurface-aliases)                                     |
+
+Examples:
+
+```Text
+sub-01/func/sub-01_task-rest_space-MNI305_bold.nii.gz
+sub-01/func/sub-01_task-rest_space-individual_bold.nii.gz
+sub-01/anat/sub-01_hemi-L_space-fsaverage5_thickness.shape.gii
+sub-01/anat/sub-01_hemi-R_space-fsaverage5_thickness.shape.gii
+sub-01/anat/sub-01_hemi-L_space-individual_thickness.shape.gii
+sub-01/anat/sub-01_hemi-R_space-individual_thickness.shape.gii
+sub-01/func/sub-01_task-rest_space-HCPMNIfsLR32k_bold.nii
+```
+
+## Common file level metadata fields
+
+Each derivative file SHOULD be described by a JSON file provided as a sidecar or
+higher up in the hierarchy of the derived dataset (according to
+[The Inheritance Principle](../02-common-principles.md#the-inheritance-principle))
+unless a particular derivative includes REQUIRED metadata fields in which case a
+JSON file is also REQUIRED.
```

Comment:
I am currently working on a dataset with an anatomical T1w scan, on which I have run the freesurfer `recon-all` command. That creates a directory full of files:

```Text
surfer/sub-01$ tree . -L 2
.
└── recon-all
    ├── bem
    ├── label
    ├── mri
    ├── scripts
    ├── src
    ├── stats
    ├── surf
    ├── tmp
    ├── touch
    └── trash

```

Would it make sense in this case to have a `surfer/sub-01/recon-all.json` file in which I specify the metadata such as the `SpatialReference` pointing to my T1w scan?